source/core.py

```diff
@@ -291,18 +291,21 @@ def handleScreenOrientationChange(self, lParam):
 			import winUser
 			# Resolution detection comes from an article found at https://msdn.microsoft.com/en-us/library/ms812142.aspx.
 			#The low word is the width and hiword is height.
-			if (winUser.LOWORD(lParam) > winUser.HIWORD(lParam):
-				if self.orientationCache == self.ORIENTATION_LANDSCAPE:
+			width = winUser.LOWORD(lParam)
+			height = winUser.HIWORD(lParam)
+			if width > height:
+				#Check to see if the height and width are the same. If so, it's actually a screen flip, and we want to alert of those!
+				if self.orientationCache[0] == self.ORIENTATION_LANDSCAPE and self.orientationCache[1:] != (width,height):

```

Comment:
While I understand you're trying to avoid unnecessary variables here, you end up building a new tuple with `[1:]` anyway. I suggest splitting out the orientation state cache (portrait/landscape) from the orientation width height cache (x, y); e.g. `self.orientationStateCache`, `self.orientationXYCache`. That also means you only have to set the width/height cache in one place, outside the condition.